benchmark/C/Savina/bndbuffer/ProdConsBenchmark.lf

```diff
@@ -182,7 +183,7 @@ reactor ManagerReactor(bufferSize:size_t(50), numProducers:size_t(40), numConsum
     =}
     
     reaction(consume) -> outConsumers, outProducers {=
-        printf("This reaction is not invoked.\n");
+        printf("consume reaction is not invoked.\n");
```

Comment:
Shouldn't this be "is invoked"?

Reply:
Looks like debug code. I'll remove them, but these benchmarks still need to be scrutinized, anyway...